feat(xo-server/xo-web/pool): avoid RPU/ host reboot, shutdown / host agent reboot during backup (#6232)
vatesfr/xen-orchestra: CHANGELOG.unreleased.md

> Users must be able to say: “Nice enhancement, I'm eager to test it”

> - [RPU/Host] Ask for confirmation when start an RPU, shutdown/reboot an host or restart host toolstack with running backups on the pool (PR [6232](https://github.com/vatesfr/xen-orchestra/pull/6232))

Reviewer: ```suggestion
- [RPU/Host] If some backup jobs are running on the pool, ask for confirmation before starting an RPU, shutdown/rebooting a host or restarting a host's toolstack (PR [6232](https://github.com/vatesfr/xen-orchestra/pull/6232))
```
